lib/db.c

```diff
@@ -7242,6 +7237,9 @@ grn_obj_get_accessor(grn_ctx *ctx, grn_obj *obj, const char *name, unsigned int
           next_obj_id = obj->header.domain;
           if (!next_obj_id) {
             // ERR(GRN_INVALID_ARGUMENT, "no such column: <%s>", name);
+            if (obj_is_referred) {
+              grn_obj_unref(ctx, obj);
+            }
```

Comment:
The `time_classify_day(timestamp-36000000000)` case only needs this.

Reply:
All other cases are also need only this.
Do we really need other changes?

Reply:
Yes, they need only this change.
However, I thought it was better to also fix other not `unref` cases because it was the same bug in the same function.

Reply:
Can you create tests for other changes?

Reply:
I guess they are tested at the CI tests for the case `GRN_ENEBLE_REFERENCE_COUNT=yes`.

Reply:
How can I confirm it?

Reply:
If tests for other cases already exist, CI with `GRN_ENABLE_REFERENCE_COUNT=yes` is red.

Reply:
I mean if CI with `GRN_ENABLE_REFERENCE_COUNT=yes` is not red, at least we don't `unref` overly.

Reply:
Ok, it is not enough to check CI with GRN_ENABLE_REFERENCE_COUNT=yes is not red,
I have separated the other changes into another PR and I will work on this to add test them.

#1335